enterprise/utils.py

```diff
@@ -1446,10 +1445,7 @@ def get_platform_logo_url():
     Return an absolute URL of the platform logo using the branding api
     """
     # Return fake URL for tests rather than mock get_logo_url for every test using EnterpriseCustomer
-    return urljoin(
-        settings.LMS_ROOT_URL,
-        get_logo_url()
-    ) if get_logo_url else 'http://fake.url'
+    return get_logo_url() if get_logo_url else 'http://fake.url'
```

Comment:
Can we make this method a little robust using `urlparse`?
here is what i mean. 

```
from urllib.parse import urlparse


def is_valid_url(url):
  try:
    result = urlparse(url)
    return all([result.scheme, result.netloc])
  except ValueError:
    return False

fake_url = 'http://fake.url'
logo_url = get_logo_url() 
if not logo_url:
    return fake_url

is_abs_url = is_valid_url(logo_url)
if is_abs_url:
    return logo_url

return urljoin(settings.LMS_ROOT_URL, logo_url)
```

the reason I say that there is a good chance that a logo can be overridden using site_configuration with either `logo_image_url` key or `university` key. so in that case, we can still get a relative URL (which would need urljoin).

Reply:
if we do this way, then this PR shouldn't be blocked by the edx-platform PR.